README.md

```diff
@@ -89,6 +89,39 @@ This works for arrays as well, for example the list of enabled render states.  H
 
 ![Render states enable](images/StatesEnable.png)
 
+### &bull; Animation editor
+
+Press <kbd>ALT</kbd> + <kbd>i</kbd> on an animation sampler to import values into the glTF JSON.
+
+```json
+"samplers": [
+{
+    "input": 6,
+    "interpolation": "LINEAR",
+    "output": 7,
+    "extras": {
+    "vscode_gltf_input": [
+        1.25,
+        2.5
+    ],
+    "vscode_gltf_type": "VEC4",
+    "vscode_gltf_output": [
+        0,
+        0,
+        0,
+        -1,
+        1,
+        0,
+        0,
+        4.4896593387466766e-11
+    ]
+},
+```
+
+Modify the `vscode_gltf_input` and `vscode_gltf_output` arrays for your needs and then press <kbd>ALT</kbd> + <kbd>o</kbd> to rewrite the glTF buffer and JSON with new values. **Warning** this will overwrite the existing binary buffer and can not be undone. Save the edited document and use `glTF: Preview 3D Model` to view your changes.
```

Comment:
The other thought I had here is create a new file name for the buffer each time.  This would allow undo/redo to function as expected but would dump a lot of garbage files in the asset path. I looked for undo/redo overrides and didn't see much; maybe I'll open a feature request so we can rerun the edits that were made.

Reply:
Interesting.  I do like the idea of leaving the original buffer intact, and adding new buffers for hand-edited data.  In fact among my own files I've got a few glTFs where I've hand-edited the JSON and saved it with a modified name, still containing a reference to the original bin file.  That sort of thing results in a folder with a single bin file shared by several similarly-named glTFs with different settings/materials/node transforms/etc.  In my case it's mostly for debug/test, but I wonder if users also have multiple glTFs in one folder referencing a shared bin file.

Also, we could advise users that if they end up with some of these "separate" animation bin files, they can export to GLB to bring all the binary data together.  In fact they can even re-import from the exported GLB to get a new single bin file with their hand-edits embedded.